lib/article/index.js

```diff
@@ -5,7 +5,7 @@ require('./style.css');
 export default class Article extends Component {
   render () {
     return (
-        <div className='articleContainer'>
+        <div className='articleContainer' onClick={() => {window.scrollTo(0, 0);this.props.viewArticle(this.props.content)}}>
```

Comment:
window.scrollTo(0,0) is just to move scroll to top, if not used, the article is displayed, but the scroll is not at the top

making it more readable, thanks